### ISSUE TITLE
Search tags

### ISSUE THREAD

**Issue by cjdduarte** [glutanimate/popup-dictionary#9]
## Prerequisite checklist

* [X] In case of a bug: Have you tried restarting Anki?
* [X] Are you running the latest version of the add-on. Have you redownloaded the add-on from AnkiWeb to make sure?
* [X] Did you check the add-on documentation (e.g. AnkiWeb description, Wiki if it exists) for known issues?
* [X] Did you perform a cursory search through the existing issue reports?

## What is the problem/feature you would like to see fixed/implemented?

I have some templates that show tags on the screen.
I can trigger the pop-up from the text of the tag, but the search is not done in the tags of the other cards.
This could be parameterized. If I want the search to be done in the tag as well.

**Comment by glutanimate** [comment]
Good idea, Carlos! Tag matches might even be more informative than phrase matches. Added to the TODO-list.

**Comment by cjdduarte** [comment]
Excellent addon. I do not know how I had not seen him before. 👍

**Comment by glutanimate** [comment]
To be fair: It was meant to be a small beta test, then people somehow found the repo and it took off from there.
